src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -539,11 +604,14 @@ const useGroupInfo = (channelId = null) => {
     setUploadedImage,
     isUploadingImage,
     setIsUploadingImage,
+    isUpdatingName,
     username,
     createChat,
     countUser,
     getMembersList,
-    handleOnNameChange,
+    handleOpenNameChange,
+    handleSaveNameChange,
+    closeOnNameChange,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props have been added to the returned object. Make sure these are used properly in the component where this hook is used.
```diff
+ isUpdatingName,
+ handleOpenNameChange,
+ handleSaveNameChange,
+ closeOnNameChange,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->